manim/mobject/geometry/labeled.py

```diff
@@ -160,3 +217,155 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         super().__init__(*args, **kwargs)
+
+
+class LabeledPolygram(Polygram):
+    """Constructs a polygram containing a label box at its pole of inaccessibility.
+
+    Parameters
+    ----------
+    vertex_groups
+        Vertices passed to Polygram constructor.
```

Comment:
A reference to the `Polygram` superclass would be great for our documentation:
```suggestion
        Vertices passed to the :class:`~.Polygram` constructor.
```